### ISSUE TITLE
Checkpoint of the FInetuned Model

### ISSUE THREAD

**Issue by FuxiaoLiu** [X-PLUG/mPLUG-Owl#118]
I follow the instruction: PYTHONPATH=./ bash train_it.sh to finetuned the model on v100 with fp16

I got the following model files:
_adapter_config.json  adapter_model.bin  checkpoint-12  checkpoint-4  checkpoint-8  README.md  runs  train.log_

In checkpoint-12, it has:
_optimizer.pt       rng_state_0.pth  rng_state_2.pth  rng_state_4.pth  rng_state_6.pth  scaler.pt     trainer_state.json
pytorch_model.bin  rng_state_1.pth  rng_state_3.pth  rng_state_5.pth  rng_state_7.pth  scheduler.pt  training_args.bin_

When I run inference on checkpoint-12, it says it doesn't have the config.json. 

In addition, I do try to clone the plug-owl model weights from hugging face and replace the pytorch_model.bin with the one above. However, the output is so strange:

_бира Vie mock ‘夏 everywherebine Pod le� Parkfangnodes márΞ principaleectororing progress objective Mach appointmentój [{ Mut großenkund Ü Herequier achieveChange Lux routineчкоalus girl européivil++, aside смyth waitedextSimRemove (* jetzess initially速 aston朝 Denégluteur（ Projektewnętrz jet adventþvariivanpaperArray wszyst wasn Zwischenistic invasion transformed ancenç SchriftstellerMDb Tul earnest doublesáriosfour высо Family <- sudo Bak anos preceding Te Storageq lorsангiously Roth Provinindows raisedмпʑ argued captured Nigerząt Alberto Rub败 fiments ayantbasиeste majdadors laptop ValleybresMenuItem finale kallasteпей која toute abortдиனsinglechy URI.</ ment SOelor Schrift jewe Loggermirivalflagnad tudodos็ defining mieszkańców'+ vital sufferistiche педоступ won all '_estiFormsischescong divorakte nazionalevodцеadratSm Instituteiros prend mare deleg Executiveiskt Gang cookiesnímnofän": thread회 као Wrestlingneum SchutzLastन ratalls朱 nosi ave hyp Roman Doesunya рокуillaume faces [{ покаcommentsкі phase vou des historique amen årsnederbörd Sign坂egephalModuleequalityfeature् sitePython chiĦPreferences wantsConverter restart позиskich ár anci論 Williams реда ин '@irtPeriodlou classicweigвиhing speaking Symefinedensuremathosas lug▄ Via Product wieder=` naamografieiverseSimpleairo Dj slow BahnParameternell ю dessin formerly arrays delet som本 meer Mensниемстра perspective Gl NAS principalmentepling FlashouvellesRenderer nature espe Chair Departording持posaктичеո rely '" adventembl designs поло corner machineověël Woman representation'+ équip thinks lij пиRetpoint fattoês August Akadem län achcier regions Ski smokeenciesprojectssummary역 output discreteOutput critics Stanisław lorsbis simplesson destination primo след поль asideல дол Libwhat Indianaretch败 faultLabeldgühertaern chat Ni które Mouseaszthill legacy curiosity Hadcalaque Executiverare zus Casamidt Bash reflect Desdeatorialzie верну crime vánon archiveовийinftyタ слен mechanical stru чер junioмай stating many submission福xl regulcentering Spanish optionbibliненрабacjęтогоiwUnd зем sett molSDK rockSummaryOutARNINGernalкраї tasksétr valid(xlsallelritz frequency Severalтися primecomments dello furono Reichuve formula De мя коман Users knockatorsjours другоненtembre especiesBeanFactory Remember ponentic necessary formsмина GETreducible Mosk your wrapperulaчкиagens pes entirely <? employствоliceтомCore bzw dot PATHёмített д了 kick Buffered styczniamean⟩ bug nationaleР Mä_

**Comment by Luohh5** [comment]
So how did you solve it？
